guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -41,51 +42,102 @@ interface ICommonControlPropertyGridComponentProps {
     onPropertyChangedObservable?: Observable<PropertyChangedEvent>;
 }
 
+
+
 export class CommonControlPropertyGridComponent extends React.Component<ICommonControlPropertyGridComponentProps> {
     constructor(props: ICommonControlPropertyGridComponentProps) {
         super(props);
     }
 
+    updateAlignment(alignment: string, value: number) {
+        const control = this.props.control;
+        if (control.typeName == "TextBlock") {
+            (this.props.control as any)["text" + alignment.charAt(0).toUpperCase() + alignment.slice(1)] = value;
+        }
+        else {
+            (this.props.control as any)[alignment] = value;
+        }
+        this.forceUpdate();
+    }
+
+    checkAndUpdateValues(propertyName : string, value:string)
+    {
+        //check if it contains either a px or a % sign
+        let percentage = false;
+        if(value.charAt(value.length-1) == '%') {

```

Comment:
supernit... `===`

Reply:
supersupernit: missing a space between `if` and `(`, as well as some spaces between `length`, `-` and `1`